tests/e2e/constants/TestConstants.ts

```diff
@@ -34,6 +34,10 @@ export const TestConstants: any = {
      */
     TS_SELENIUM_BASE_URL: getBaseUrl(),
 
+    TS_SELENIUM_OPENSHIFT_CONSOLE_URL(): string {
```

Comment:
The constant looks redundant, because I can't remember other OCP 4.x application name, but `console-openshift-console`.

Reply:
sorry, I didn't get, it exactly function that makes ocp link from base with replacement "decspaces" to "console-openshift-console". Should I move this method somewhere?

Reply:
TestConstants.ts holds test constants which can be redefined using environment variables.

`TS_SELENIUM_OPENSHIFT_CONSOLE_URL():` is not constant, and doesn't look like method with capitalized characters.

Taking into account that it's needed for `LoginTests.loginIntoOcpConsole()`, I would suggest to move the logic to there.

Reply:
Makes sense

Reply:
Thanks